### DISCUSSION TITLE
What is UTXO?

### DISCUSSION THREAD

**Discussion by abharun** [abharun/mini-chain#10] [Q&A] [answered]
What is UTXO? Why is it need even with transactions?

**Comment by logidexp** [comment, marked as answer]
UTXO is a concept in POW network.
It only has output of amount, and the current amount of account is calculated by adding all the UTXOs for the account.
When a TX is triggered, the network searches the UTXOs for the sending account.
Then it erases the UTXOs till the required amount is all transferred, and if there's remaining UTXO, new UTXO is created.
For example:
current UTXOs: [30, 40, 20, 60]
transfer amount: 35
then the first UTXO is erased, and for the remaining amount 5, UTXO 40 is erased and new UTXO with 35 is created.
So the result is : [20, 60, 35]
